### PR TITLE
Add ability to run tests in seeded-random order through `--seed` flag

### PR DESCRIPTION
**Pull request by Arkni**
Addresses #124

### CODE REVIEW

**Comment by Arkni** [comment]
@jzaefferer  @vladikoff  r?

File: docs/qunit-options.md [hunk 1]
```diff
@@ -54,8 +54,20 @@ Default: `false`
...
-`grunt qunit --modules=foo` will run the module `foo`. You can specify one or multiple, comma-separated modules to run.
+`grunt qunit --modules="foo"`
+
+Will run the module `foo`. You can specify one or multiple, comma-separated modules to run.
+
+## Running tests in seeded-random order: `--seed`
+
+`grunt qunit --seed="a-string"`
+
+Tells `QUnit` to run tests in a seeded-random order. The value provided will be used as the seed in a pseudo-random number generator to ensure that results are reproducible. The randomization will also respect the reorder option if enabled and re-run failed tests first without randomizing them.
```
Comment (jzaefferer) [inline docs/qunit-options.md]: This seems like it should "just" link to the right QUnit docs instead. At least shorten these paragraphs?

**Comment by jzaefferer** [comment]
Looks good to me.

File: docs/qunit-options.md (hunk 1 above)
Comment (Arkni) [inline docs/qunit-options.md]: Make sense. I'll update this and merge the PR as soon as possible.

Thanks!

Comment (Arkni) [inline docs/qunit-options.md]: How about this wording?
```md
Specify the seed to pass to QUnit. See [`QUnit.config.seed`](https://api.qunitjs.com/QUnit.config/) docs for more information.
```

Comment (jzaefferer) [inline docs/qunit-options.md]: `Specify the seed to pass to QUnit, to run tests in random, but deterministic order. [...]`?

Comment (Arkni) [inline docs/qunit-options.md]: Nice :+1:

**Comment by Arkni** [comment]
Updated and merged 461e408

Thanks @jzaefferer :)
